monero-outreach-docs/en/PSA/files/learning/learning_en.sla

```diff
@@ -68,56 +68,56 @@
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="getmonero.org (Of"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" KERN="0" CH="f"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" KERN="-15" CH="i"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="11" FEATURES="inherit underline" CH="cial Website)"/>
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="432" YPOS="568.5" WIDTH="160" HEIGHT="162.75" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFEATURES="inherit underline" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="MoneroSerif Italic" ISIZE="11" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="0" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 160 0 160 0 160 0 160 0 160 162.75 160 162.75 160 162.75 160 162.75 0 162.75 0 162.75 0 162.75 0 162.75 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 160 0 160 0 160 0 160 0 160 162.75 160 162.75 160 162.75 160 162.75 0 162.75 0 162.75 0 162.75 0 162.75 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FONT="MoneroSerif Italic" FONTSIZE="11" FEATURES="inherit underline" CH="getmonero.org (Of"/>
```

Comment:
This font has not been fixed yet. [MoneroSerif-Bolditalic.otf](https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/fonts/monero%20fonts/MoneroSerif_v2/MoneroSerif-BoldItalic.otf)

Reply:
Fixed. The forks files aren't supposed to contain the bold italic font if I'm not wrong.